build/create-safe-same-address-different-network.md

```diff
@@ -0,0 +1,86 @@
+
+1. If you are recreating a Safe created on Ethereum's Mainnet, the Safe will be created but it won't show up on the Safe UI. You will have to load it using another tool like the Safe See Appendix. [Recreating a Safe from Ethereum Mainnet](#recreating-a-safe-from-ethereum-mainnet) for more information.
+
+## Use Metamask
+
+The easiest way to recover your Safe is by using Metamask.
+
+See also: [Recreating a Rinkeby Safe on Polygon](https://www.loom.com/share/ca34aabcd62747fb9fb89bd463b4c741)
+
+### Enable Show Hex Data in Metamask
+![Screen Shot 2023-03-17 at 4 00 39 PM](https://user-images.githubusercontent.com/9806858/226020125-788ae92c-9f98-486a-89fb-45e88da179ec.png)
+
+### Add Network
+
+![Screen Shot 2023-03-17 at 4 00 53 PM](https://user-images.githubusercontent.com/9806858/226020214-56e5e9c2-9b7f-4cd0-b5e5-a21c73f91d88.png)
+
+### Verify [Proxy Factory has the same address on Both Chains](https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/proxy_factory.json)
+![Screen Shot 2023-03-17 at 4 08 07 PM](https://user-images.githubusercontent.com/9806858/226023682-4855ce25-129d-47b1-9f24-3b74a12852fb.png)
+
+### Alternate: Check the [List of deployed Proxy Factory Addresses on Other Chains](https://blockscan.com/address/0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2)
+
+![Screen Shot 2023-03-23 at 7 34 31 AM](https://user-images.githubusercontent.com/9806858/227191742-60aaab59-be71-4d8f-9f21-1a57d1910e0c.png)
+
+
+### Get [Transaction Creation Data](https://app.safe.global/transactions/history?safe=gor:0xF188d41FD181f94960C5451D7ff6FdbcDf201a71)
+
+![Screen Shot 2023-03-23 at 7 33 48 AM](https://user-images.githubusercontent.com/9806858/227191393-21843e91-0ac5-457c-9133-6a6d317ef8aa.png)
+
+
+### Copy Safe Contract Creation Data
+
+![Screen Shot 2023-03-17 at 4 06 53 PM](https://user-images.githubusercontent.com/9806858/226022160-2907617e-6283-41c1-bc97-ccd90d8ff9ef.png)
+
+### Paste Proxy Factory and Transaction Hex into Metamask
+
+![Screen Shot 2023-03-17 at 4 04 08 PM](https://user-images.githubusercontent.com/9806858/226022325-4be63630-9476-4d4e-aef7-498966f3bfa8.png)
+
+
+## Part 3: Recreating Safe with Safe {Core} SDK
+
+See the code snippet in [`recreateSafe.ts`](/examples/utils/recreateSafe.ts). 
+
+## Appendix
+
+### How it Works
+
+Safes are created by calling [`createProxyWithNonce` which uses `CREATE2`](https://github.com/safe-global/safe-contracts/blob/ba526d0475e3004f9fcd71cd25ebbd225ebeee7f/contracts/proxies/SafeProxyFactory.sol#L32)
+
+The [CREATE2 opcode](https://eips.ethereum.org/EIPS/eip-1014) gives you the ability to predict the address where a contract will be deployed, without deploying it. See [Deploying Smart Contracts Using CREATE2](https://docs.openzeppelin.com/cli/2.8/deploying-with-create2) for more information about this.
+
+### Recreating a Safe from Ethereum Mainnet
+
+If you recreate a Safe that was originally created on Ethereum Mainnet not another chain, it won't show up in the UI.
```

Comment:
There are other cases where this is also the case. In general if the Safe does not use the singleton that we support on that specific chain. (i.e. on optimism there are multiple versions deployed, but we do not support all of them via our interface)